kubernetes/overlays/prod/prompt-proto-service/prompt-proto-service.yaml

```diff
@@ -26,14 +26,18 @@ spec:
           value: LATISS
         - name: PIPELINES_CONFIG
           value: >
-            (survey="AUXTEL_PHOTO_IMAGING")=${PROMPT_PROTOTYPE_DIR}/pipelines/${RUBIN_INSTRUMENT}/ApPipe.yaml
-            (survey="spec")=None
-            (survey="spec_with_rotation")=None
-            (survey="spec_bright")=None
-            (survey="spec_bright_with_rotation")=None
-            (survey="spec_pole")=None
-            (survey="spec_pole_with_rotation")=None
-            (survey="")=None
+            (survey="AUXTEL_PHOTO_IMAGING")=[${PROMPT_PROTOTYPE_DIR}/pipelines/${RUBIN_INSTRUMENT}/ApPipe.yaml,
+                                             ${PROMPT_PROTOTYPE_DIR}/pipelines/${RUBIN_INSTRUMENT}/SingleFrame.yaml,
+                                             ${PROMPT_PROTOTYPE_DIR}/pipelines/${RUBIN_INSTRUMENT}/Isr.yaml]
+            (survey="AUXTEL_DRP_IMAGING")=[${PROMPT_PROTOTYPE_DIR}/pipelines/${RUBIN_INSTRUMENT}/SingleFrame.yaml,
```

Comment:
I think we also want to try ApPipe first for `"AUXTEL_DRP_IMAGING"` as well.

Reply:
I thought `_DRP_` rarely or never repeats fields? See [this Slack thread](https://lsstc.slack.com/archives/C02HPBY4ZFC/p1689280421436659).

Reply:
Yes, I agree with that thread, but that is also part of the motivation for this feature. We can try AP and fall back on single frame processing if necessary. If we couldn't fall back, we would certainly only want to configure it to run just single frame.